views/result/account.validation.pending.html

```diff
@@ -1,29 +1,20 @@
-{% set html %}
-    <div class="govuk-width-container">
-        <div class="gds-loading-icon-container">
-            <div class="spinner"></div>
-        </div>
-        <h1 class="govuk-heading-l .govuk-text-align-centre">Validation in progress</h1>
-        <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-8 .govuk-text-align-centre"><span id="percentage">0</span>% complete</p>
-        <p class="govuk-body .govuk-text-align-centre">This page will automatically update when the validation is complete.</p>  
+<div class="govuk-width-container">
+    <div class="gds-loading-icon-container">
+        <div class="spinner"></div>
     </div>
+    <h1 class="govuk-heading-l .govuk-text-align-centre">Validation in progress</h1>
+    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-8 .govuk-text-align-centre"><span id="percentage">0</span>% complete</p>
+    <p class="govuk-body .govuk-text-align-centre">This page will automatically update when the validation is complete.</p>  
```

Comment:
The class names shouldn't have a `.` infront. 
```suggestion
    <h1 class="govuk-heading-l govuk-text-align-centre">Validation in progress</h1>
    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-8 govuk-text-align-centre"><span id="percentage">0</span>% complete</p>
    <p class="govuk-body govuk-text-align-centre">This page will automatically update when the validation is complete.</p>  
```

Reply:
Thanks sorted the typo